extension/install.rdf

```diff
@@ -6,7 +6,8 @@
     <em:name>Personas Plus</em:name>
     <em:version>1.7.7</em:version>
     <em:description>Your Firefox, your style. Easy-to-change themes.</em:description>
-
+	<em:bootstrap>true</em:bootstrap>
+	<em:unpack>true</em:unpack>
```

Comment:
I don't see a reason for this to be unpacked and it can cause some troubles with signing, so let's not do that.